### DISCUSSION TITLE
Help with "INFO: Could not find files for the given pattern(s) "

### DISCUSSION THREAD

**Discussion by HoopThe3rd** [jef/streetmerchant#1595] [Help] [answered]
Hello, I'd like to start by saying that I read through similar issues but they ended by changing the file from a .txt to a .env. I'm 99% sure that's not my problem which is why I'm posting here. Below is my File Folder, .env file, and terminal log. I had to switch the .env file to text for upload but it is currently set as a .env.
[ENV.txt](https://github.com/jef/streetmerchant/files/5766352/ENV.txt)
![image](https://user-images.githubusercontent.com/61106533/103569706-a0770d00-4e7c-11eb-8bcd-0c5a5767171c.png)
![image](https://user-images.githubusercontent.com/61106533/103569791-cdc3bb00-4e7c-11eb-9c0f-b8658c373e46.png)

**Comment by jef** [comment, marked as answer]
It looks like `WEB_PORT` is set to something that's not between 0 and 65536. You can also ignore that Could not find files for given pattern(s) message.

**Reply by HoopThe3rd** [reply]
Thanks that fixed it

**Reply by jef** [reply]
Great! Good luck 🚀
